addons/crm_sms/views/crm_lead_views.xml

```diff
@@ -26,4 +26,26 @@
             'default_res_id': active_id,
         }"
     />
-</odoo>
\ No newline at end of file
+
+    <record id="crm_case_tree_view_my_activities_extend" model="ir.ui.view">
+        <field name="name">crm.lead.tree.sms.button</field>
```

Comment:
BUT name contains more details, it could be crm.lead.view.list.activities.inherit.sms